src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,54 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    private function getBundleData()
+    {
+        $bundles = array();
+
+        try {
+            $composerLockPath = $this->kernel->getRootDir().'/../composer.lock';
+            $composerLock = json_decode(file_get_contents($composerLockPath), true);
+
+            foreach (array_merge($composerLock['packages'], $composerLock['packages-dev']) as $dependency) {
+                $installedDependencies[$dependency['name']] = $dependency['version'];
+            }
+        } catch (\Exception $e) {
+            $installedDependencies = array();
+        }
+
+        foreach ($this->kernel->getBundles() as $name => $bundle) {
+            $bundlePath = $bundle->getPath();
+
+            $bundleVersion = null;
+            foreach ($installedDependencies as $packageName => $packageVersion) {
+                if (preg_match(sprintf('~.*/vendor/%s~', $packageName), $bundlePath)) {
```

Comment:
does it work fine on Windows here ? I'm not sure ``$bundle->getPath()`` normalizes separators